lib/getPlantUMLDiagram.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Generates plantUML class diagram from default output syntax
+ * 
+ * @param {Object} metrics Relations between AsyncAPI docs in default output syntax
+ * @returns {String} class diagram following plantUML syntax
+ */
+function getPlantUMLDiagram(metrics) {
+  let i=1;
+  let plantUMLSyntax = '';
+  metrics.forEach((relations,server) => {
+    const [url, protocol] = server.split(',');
+    plantUMLSyntax+= `\nclass server${i} { \n url: ${url} \n protocol: ${protocol}\n}`;
+    
+    relations.forEach((operations, channel) => {
+      operations.sub.forEach((metadata,subscriber) => {
+        const service = subscriber.replace(/\s/g,'');
+        plantUMLSyntax+=`\n${service} --|> server${i}:${channel}`;
+      });
+    
+      operations.pub.forEach((metadata,publisher) => {
+        const service = publisher.replace(/\s/g,'');
+        plantUMLSyntax+=`\nserver${i} --|> ${service}:${channel}`;
+      });
+    });
+    i++;
```

Comment:
I can't see what this is for. Did you have a classic for loop before?

Reply:
`i` is needed, because `Map.prototype.forEach` hasn't in callback arguments the index. 

@arjungarg07  We can use the name of server, but as we discussed in previous PRs this won't to be good, because the names of the servers between docs may change.